lib/pure/distros.nim

```diff
@@ -273,7 +273,7 @@ proc foreignDep*(foreignPackageName: string) =
   let (installCmd, sudo) = foreignDepInstallCmd(foreignPackageName)
   foreignCmd(installCmd, sudo)
 
-proc echoForeignDeps*() =
+proc echoForeignDeps*() {.deprecated: "use `foreignDeps` instead".} =
```

Comment:
So... instead I have to use the very same loop that `echoForeignDeps` previously offered me? Come on, if you want to export `foreignDeps` just do that instead of a crazy crusade in the name of "minimalism".

Reply:
No.
Just `echo foreignDeps`. 🤷